client/js/models/archive.js

```diff
@@ -276,9 +241,7 @@ module.exports = {
         promises.push(promise)
       } else { // download entire archive as zip
         zipName = state.key
-        var keys = Object.keys(state.entries.sort())
-        keys.forEach((key, i) => {
-          const entry = state.entries[key]
+        state.entries.forEach((entry) => {
```

Comment:
there wasn't a reason except it made for easier testing but we aren't testing.

Also there are no keys anymore, it's an array now so this doesn't do anything